tests/functional/u/unpacking_non_sequence.py

```diff
@@ -137,3 +138,12 @@ def flow_control_unpacking(var=None):
         var0, var1 = var
         return var0, var1
     return None
+
+
+class V2(NamedTuple):
+    x: float
+    y: float
+
+    def sum(self):
+        x, y = self
+        return x + y
```

Comment:
```suggestion
        return x + y
        
   def sum_3d(self):
        x, y, z = self
        return x + y + z        
```
Wondering what happens when we try to unpack more than what we can.

Reply:
This can probably be resolve. It will emit `unbalanced-tuple-unpacking`, see test case above.